js/actions/test.js

```diff
@@ -26,7 +26,9 @@
 
 const Parse = require('parse/react-native');
 const ActionSheetIOS = require('ActionSheetIOS');
+const Platform = require('Platform');
 const {version} = require('../env');
+const SendIntentAndroid = Platform.OS === 'android' ? require('react-native-send-intent') : undefined;
```

Comment:
I take that back, sorry :) Filed https://github.com/lucasferreira/react-native-send-intent/issues/19. This is good to go, I'll merge it tomorrow. Thank you!